code/modules/vore/fluffstuff/custom_guns_vr.dm

```diff
@@ -852,4 +852,4 @@
 		list(mode_name="lethal", fire_delay=12, projectile_type=/obj/item/projectile/beam, modifystate="holdoutkill", charge_cost = 600),
 		list(mode_name="low-power", fire_delay=8, projectile_type=/obj/item/projectile/beam/weaklaser, modifystate="holdoutstun", charge_cost = 120),
 		list(mode_name="stun", fire_delay=12, projectile_type=/obj/item/projectile/beam/stun/med, modifystate="holdoutshock", charge_cost = 300),
-	)
```

Comment:
Can you revert this file entirely? Also that's the _vr not the _yw custom guns file.